docs/install_guides/installing-openvino-from-archive-windows.md

```diff
@@ -39,74 +38,72 @@ You can also check the [Release Notes](https://software.intel.com/en-us/articles
 
 ### <a name="install-openvino"></a>Step 1: Download and Install OpenVINO Core Components
 
-1. Select and download the OpenVINO™ archive files from [Intel® Distribution of OpenVINO™ toolkit download page](https://software.intel.com/en-us/openvino-toolkit/choose-download).
-   There are typically two files for you to download: 
-   ```sh
-   w_openvino_toolkit_<operating system>_<release version>_<package ID>_x86_64.zip
-   w_openvino_toolkit_<operating system>_<release version>_<package ID>_x86_64.zip.sha256
-   ``` 
-   where the `.sha256` file is used to verify the success of the download process.
-   
-2. Locate the downloaded files in your system. This document assumes the files are in your `Downloads` directory. 
-   
-3. Open a command prompt terminal window, and run the following command with the `.zip` file:
-   ```sh
-   CertUtil -hashfile <archive name>.zip SHA256
-   ```
-   Compare the returned value in the output with what's in the `<archive name>.zip.sha256` file:
-   * If the values are the same, you have downloaded the correct file successfully.
-   * If not, create a Support ticket [here](https://www.intel.com/content/www/us/en/support/contact-intel.html).
+First, open a command terminal as administrator by searching for "Command Terminal" in the Start menu, right clicking on it, and selecting "Run as administrator". Create a folder for OpenVINO and move into it by issuing the following commands.
 
-4. Unzip the `<archive name>.zip` file using your preferred archive tool. 
-   > **NOTE**: The name of the archive file might be long. Make sure that you have [enabled long paths on Windows](https://docs.microsoft.com/en-us/windows/win32/fileio/maximum-file-path-limitation?tabs=registry#enable-long-paths-in-windows-10-version-1607-and-later) to avoid possible errors in the unzipping process.
+```sh
+mkdir "C:\Program Files (x86)\Intel"
+cd "C:\Program Files (x86)\Intel"
+```
 
- 
-The standard OpenVINO `<INSTALL_DIR>` used in this document is `C:\Program Files (x86)\Intel\openvino_<version>\`. You're recommended to move the extracted files to that directory. 
+> **NOTE**: C:\Program Files (x86)\Intel is the recommended folder path, but you may use a different path if desired or if you don't have administrator priveleges on your PC. If the C:\Program Files (x86)\Intel folder already exists, skip the `mkdir` command.
 
-For simplicity, you can also create a symbolic link to the latest installation: `C:\Program Files (x86)\Intel\openvino_2022\`.
+Next, you'll download the OpenVINO Runtime 2022.2 archive file from the [OpenVINO archives](https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.2/windows) site. Issue the following command to download archive file, unzip it, and rename the folder to `openvino_2022.2.0.7713`:
+```sh
+curl https://storage.openvinotoolkit.org/repositories/openvino/packages/2022.2/windows/w_openvino_toolkit_windows_2022.2.0.7713.af16ea1d79a_x86_64.zip --output openvino_2022.2.0.7713.zip
```

Comment:
should we point to github archives? They have download statistics

Reply:
Yes, we could do that. What is the link to the GitHub archives?

Reply:
Please, find it here https://github.com/openvinotoolkit/openvino/releases/tag/2022.2.0

Reply:
Thanks Ilya! I changed it to use the GitHub link instead.